.gitignore

```diff
@@ -345,4 +345,5 @@ ASALocalRun/
 healthchecksdb
 
 # lib/nuget
-lib/
\ No newline at end of file
+lib/
+/EasyCommands_Backup_2022-01-07-144211726.zip
```

Comment:
Let's remove this, looks like you can delete locally

Reply:
see above ...